src/routers/projects.js

```diff
@@ -2,8 +2,91 @@ const express = require("express");
 
 const router = express.Router();
 
-router.get("/", function (req, res, next) {
+router.get("/", function (req, res) {
   res.status(200).send({ sucess: true });
 });
 
+router.get("/:projectId/versions", async (req, res) => {
+  const { projectId } = req.params;
+  const accessToken = req.headers.authorization;
+
+  if (accessToken) {
+    try {
+      const getVersions = await fetch(
+        `https://api.figma.com/v1/files/${projectId}/versions`,
+        {
+          method: "GET",
+          headers: {
+            "X-FIGMA-TOKEN": accessToken,
+          },
+        },
+      );
+
+      const responseJson = await getVersions.json();
+      const { status, versions } = responseJson;
+
+      if (status === 400) {
+        return res.status(400).send({
+          result: "error",
+          error: {
+            message: "Bad Request",
+          },
+        });
+      }
+
+      if (status === 403) {
+        return res.status(403).send({
+          result: "error",
+          error: {
+            message: "인증되지 않은 사용자 접근입니다.",
+          },
+        });
+      }
+
+      if (status === 404) {
+        return res.status(404).send({
+          result: "error",
+          error: {
+            message: "존재하지 않는 피그마 파일입니다. 다시 입력해주세요",
+          },
+        });
+      }
+
+      if (status === 500) {
+        return res.status(500).send({
+          result: "error",
+          error: {
+            message: "프로젝트 용량이 너무 커 응답에 실패했습니다.",
+          },
+        });
+      }
+
+      if (versions.length < 2) {
+        return res.status(502).send({
+          result: "error",
+          error: {
+            message: "비교할 수 있는 버전이 없어서 서비스 이용이 불가합니다.",
+          },
+        });
+      }
+
+      return res.status(200).send(responseJson);
```

Comment:
- P1
`responseJson`이 json형식이니까
```suggestion
      return res.status(200).json(responseJson);
```
으로 수정하면 더 명시적일 것 같습니다!

<br/>

• 참고
[res.send vs res.json](https://haeguri.github.io/2018/12/30/compare-response-json-send-func/)
[Express-res.json](https://expressjs.com/en/api.html#res.json)

Reply:
JSON 데이터 전달할때 명시적인 측면이나 비용측면에서 req.json을 사용하는게 맞는것 같습니다.